constraint_utils/constraint_basics.py

```diff
@@ -635,9 +712,23 @@ def collapse_lof_ht(lof_ht: hl.Table, keys: Tuple[str], calculate_pop_pLI: bool
 def annotate_constraint_groupings(ht: Union[hl.Table, hl.MatrixTable],
                                   custom_model: str = None) -> Tuple[Union[hl.Table, hl.MatrixTable], List[str]]:
     """
-    HT must be exploded against whatever axis
-
-    Need to add `'coverage': ht.exome_coverage` here (which will get corrected out later)
+    Add constraint annotations.
```

Comment:
something like add annotations to be used for groupings would be more descriptive